codecademy/functions.py

```diff
@@ -36,7 +37,7 @@ def distance_from_zero(arg):
 # 4. Function ex-19
 def distance_from_zero_type(arg):
     """Depending on the type of argument returned absolute value or 'Nope'"""
-    if type(arg) == int or type(arg) == float:
+    if type(arg) == int or type(arg) == float or type(arg) == long:
```

Comment:
same thing here you can improve checking the type of the variable by using `isinstance()`